plugins/catalog-backend/src/ingestion/processors/GithubReaderProcessor.ts

```diff
@@ -83,11 +89,6 @@ export class GithubReaderProcessor implements LocationProcessor {
         throw new Error('Wrong GitHub URL');
       }
 
-      // Removing the "blob" part
-      url.pathname = [empty, userOrOrg, repoName, ...restOfPath].join('/');
```

Comment:
Bit confused here tbh, why does this fail if the branch name contains `/`? It seems functionally the same as the new approach.

Reply:
Good question, the issue that I took was described as "GitHubReaderProcessor cannot process branch names with "/" in them", so I started working on a solution and tested that it worked but didn't have the opportunity to test the old code to verify that it was a valid issue being reported. Maybe I'm missing something? I'll try to test the old code, but now I look at it, I concur that it shouldn't fail if the branch name contains `/`.